app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable shared among all instances of `Animal`. This is correct if you want to track all living animals across all instances, but ensure this is the intended behavior.